.github/workflows/ci.yml

```diff
@@ -0,0 +1,55 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - "**" #double star includes a '/'.  single star doesnt match a '/'
+    tags:
+      - "**" #double star includes a '/'.  single star doesnt match a '/'
+  schedule:
+    # Daily at 05:44
+    - cron: '44 5 * * *'
+
+defaults:
+  run:
+    shell: bash
+
+env:
+  PYTHON_VERSION: 3.7
+  PYTHON: c:/hostedtoolcache/windows/Python/3.7.9/x64
```

Comment:
Maybe explain where this is used and how it would be nice if it were done another way, but not bothering yet, or whatever.